integration/spark/src/test/scala/org/apache/carbondata/geo/GeoTest.scala

```diff
@@ -122,7 +161,8 @@ class GeoTest extends QueryTest with BeforeAndAfterAll with BeforeAndAfterEach {
     createTable(sourceTable)
     loadData(sourceTable)
     createTable(targetTable, "'SORT_SCOPE'='GLOBAL_SORT',")
-    sql(s"insert into  $targetTable select * from $sourceTable")
+    sql(s"insert into  $targetTable select timevalue, longitude," +
```

Comment:
why select * is changed to individual columns ? Probably because you get geospatial column as well when you give select *.  But, we need to handle this insert into select *.

Reply:
Done